airflow/operators/docker_operator.py

```diff
@@ -14,12 +14,17 @@
 
 import json
 import logging
+import ast
 from airflow.exceptions import AirflowException
 from airflow.models import BaseOperator
 from airflow.utils.decorators import apply_defaults
 from airflow.utils.file import TemporaryDirectory
-from docker import Client, tls
-import ast
+from docker import tls
+try:
+    from docker import APIClient
+except ImportError:
+    # prior to version 2.0.0 of the module docker, the API client was called Client
+    from docker import Client as APIClient
```

Comment:
Given we now depend upon the more recent docker lib (and that is the right thing to do) I don't think there is any need to have this fallback behaviour -- we will always have at least docker 2.0 won't we?

Reply:
Sure. Maybe we should have that reflect in the setup.py as well? The master version still read "docker = ['docker-py>=1.6.0']" which now should be "docker = ['docker-py>=2.0.0']"